wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-meta-info.php

```diff
@@ -5,37 +5,81 @@
+					<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size aligncenter is-style-text has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:400;line-height:0">
+						<a class="wp-block-button__link has-text-align-center wp-element-button" href="<?php echo esc_attr( $current_post->slides_view_url ); ?>" style="border-radius:2px;padding-top:16px;padding-right:13px;padding-bottom:16px;padding-left:13px" target="_blank" rel="noreferrer noopener">
+							<?php esc_html_e( 'View slides', 'wporg-learn' ); ?>
+							<span aria-hidden="true" class="wp-exclude-emoji">↗</span>
+						</a>				
+					</div>
+					<!-- /wp:button -->
+				<?php endif; ?>
+				<?php if ( $current_post->slides_download_url ) : ?>
+					<!-- wp:button {"textAlign":"center","width":100,"style":{"border":{"radius":"2px"},"spacing":{"padding":{"left":"13px","right":"13px","top":"16px","bottom":"16px"}},"typography":{"lineHeight":0,"fontStyle":"normal","fontWeight":"400"}},"className":"aligncenter is-style-text","fontSize":"normal","fontFamily":"inter"} -->
+					<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size aligncenter is-style-text has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:400;line-height:0">
+						<a class="wp-block-button__link has-text-align-center wp-element-button" href="<?php echo esc_attr( $current_post->slides_download_url ); ?>" style="border-radius:2px;padding-top:16px;padding-right:13px;padding-bottom:16px;padding-left:13px" target="_blank" rel="noreferrer noopener">
+							<?php esc_html_e( 'Download slides', 'wporg-learn' ); ?>
+							<span aria-hidden="true" class="wp-exclude-emoji">↗</span>
+						</a>				
+					</div>
+					<!-- /wp:button -->
+				<?php endif; ?>
+			</div>
+			<!-- /wp:buttons -->
+		<?php endif; ?>
+		<!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
+		<div class="wp-block-buttons">
+			<!-- wp:button {"textAlign":"center","width":100,"style":{"border":{"radius":"2px"},"spacing":{"padding":{"left":"13px","right":"13px","top":"16px","bottom":"16px"}},"typography":{"lineHeight":0,"fontStyle":"normal","fontWeight":"400"}},"className":"aligncenter is-style-fill","fontSize":"normal","fontFamily":"inter"} -->
+			<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size aligncenter is-style-fill has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:400;line-height:0">
+				<a class="wp-block-button__link has-text-align-center wp-element-button" onclick="window.print()" style="border-radius:2px;padding-top:16px;padding-right:13px;padding-bottom:16px;padding-left:13px" target="_blank" rel="noreferrer noopener">
+					<?php esc_html_e( 'Print Lesson Plan', 'wporg-learn' ); ?>
+				</a>				
+			</div>
+			<!-- /wp:button -->
+		</div>
+		<!-- /wp:buttons -->
```

Comment:
Noting that we don't have a print stylesheet, so printing these pages results in some pretty mangled header and footer elements. I think in the current theme we've set `display:none` on all of this. Out of scope for this PR obviously. How high priority is printing still @bsanevans @westnz ?